modules/client/src/internal/client/methods.ts

```diff
@@ -21,16 +21,16 @@ import { AddressZero } from "@ethersproject/constants";
 import { Contract, ContractTransaction } from "@ethersproject/contracts";
 import { erc20ContractAbi } from "../abi/erc20";
 import {
-  QueryTokenBalances,
   QueryDao,
   QueryDaos,
+  QueryTokenBalances,
   QueryTokenTransfers,
 } from "../graphql-queries";
 import {
   AssetBalance,
-  AssetBalanceQueryParams,
   AssetBalanceSortBy,
   CreateDaoParams,
+  DaoBalancesQueryParams,
```

Comment:
I see, these 3 make sense together as "asset". But assets doesn't relate to the internal usage of `Token` as you said 🤔 

Is there any quick fix to make everything consistent/coherent here?

Reply:
Without breakin what they have in the ui rn, no

Reply:
Ok, then let's keep things stable for the UI where possible :+1: